topsf/tasks/inference_v2/combine_base.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+
+import luigi
+import law
+import os
+
+from columnflow.tasks.framework.base import Requirements, CommandTask
+from columnflow.tasks.framework.mixins import (
+    CalibratorsMixin, SelectorStepsMixin, ProducersMixin, InferenceModelMixin,
+)
+from columnflow.tasks.framework.remote import RemoteWorkflow
+from topsf.tasks.inference import CreateDatacards
+from columnflow.util import dev_sandbox
+
+from topsf.tasks.inference_v2.fit_mixin import FitMixin
+
+
+class CombineBaseTask(
```

Comment:
I would suggest to rename this to something not combine-specific, since tasks inheriting from this will not necessarily run the `combine` command, but also e.g. `text2workspace`.
```suggestion
class InferenceBaseTask(
```